### PR TITLE
Add milestone schedule form page

### PR DESCRIPTION
**Pull request by mgwalker**
Adds the milestone schedule page to the form UX prototype.

### This pull request is ready to merge when...
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- ~~Design has approved the experience~~
- ~~Product has approved the experience~~

### CODE REVIEW

File: web/src/components/FormActivitySchedule.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+          <th>Actual end date</th>
+          <th />
+        </tr>
+      </thead>
+      <tbody>
+        <FieldArray name="milestones" component={Milestones} />
+      </tbody>
+    </table>
+
+    <FieldArray name="milestones" component={AddButton} />
```
Comment (brendansudol) [inline web/src/components/FormActivitySchedule.js]: Not sure I understand this bit -- why is `AddButton` being used as a FieldArray? Is that just so it gets the `fields` prop for `milestones`?

Comment (mgwalker) [reply web/src/components/FormActivitySchedule.js]: Yeah, that's exactly why.  It was the most obvious way of getting access to the `fields` prop - and faster than digging through redux-form docs.  😛

Comment (brendansudol) [reply web/src/components/FormActivitySchedule.js]: Ok, i don't love that, but we can figure out a cleaner way and refactor soon

Comment (mgwalker) [reply web/src/components/FormActivitySchedule.js]: I also don't love it.  I think this falls under the conversation we were having about how to take the `redux-form` state and merge it into our "real" state.

**Pull request merged**